spec/integration/split_csv_export_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "tmpdir"
+
+RSpec.describe "Split Smary Survey CSV Export" do
+  around do |example|
+    travel_to(Time.zone.parse("2020-05-01 10:00")) { example.run }
+  end
+
+  before do
+    Rake::Task["split_csv_export"].reenable
+  end
+
+  context "when a file is given" do
+    around do |example|
+      expect { example.run }.to output.to_stdout
+    end
+
+    it "can split a CSV export into files for each partner" do
+      csv_path = File.join(__dir__, "../support/files/live_export.csv")
+      tmpdir = Dir.mktmpdir
+
+      begin
+        ClimateControl.modify(CSV_FILE: csv_path, OUTPUT_DIR: tmpdir) do
+          Rake::Task["split_csv_export"].invoke
+        end
+
+        expect(File).to exist(File.join(tmpdir, "2020-05-01-cabinet-office.csv"))
+        expect(File).to exist(File.join(tmpdir, "2020-05-01-third-party.csv"))
+      ensure
```

Comment:
Minor: we could delegate this to [the block version of `mktmpdir`](https://ruby-doc.org/stdlib-2.5.3/libdoc/tmpdir/rdoc/Dir.html#method-c-tmpdir).

Reply:
Oh cool - hadn't noticed that.